packages/babel-core/test/api.js

```diff
@@ -48,26 +47,24 @@ describe("parser and generator options", function() {
 
   it("options", function() {
     const string = "original;";
-    assert.deepEqual(
-      newTransform(string).ast,
+    expect(newTransform(string).ast).toEqual(
       babel.transform(string, { ast: true }).ast,
     );
-    assert.equal(newTransform(string).code, string);
+    expect(newTransform(string).code).toEqual(string);
```

Comment:
`.code` is a string so imho `.toBe` would be more appropriate here

Reply:
I believe I messed up in applying the codemod (meaning I applied the code-mod to an already modified file. These were supposed to be correctly changed to `toBe` according to the code-mod (if I had applied it correctly).
Changing now.